test/test__helper.rb

```diff
@@ -34,11 +34,29 @@
 end
 
 module Minitest
+  require_relative '../lib/zold/log'
+
   class Test
     def test_log
-      require_relative '../lib/zold/log'
-      # @test_log = Zold::Log::Quiet.new
-      @test_log ||= Zold::Log::Verbose.new
+      @test_log ||= verbosity_by_env_log
+    end
+
+    def verbosity_by_env_log
+      env_var_truthy?('TEST_QUIET_LOG') ? quiet_log : verbose_log
+    end
+
+    private
+
+    def env_var_truthy?(var_name)
+      ENV[var_name] && ENV[var_name] != 'false'
+    end
+
+    def quiet_log
```

Comment:
@mgrassotti I don't understand why we need methods for that? Can't we just use objects in the line no.45 above? Why making methods? Every method is an additional maintainabiliity problem. Try to keep the code as short a possible.

Reply:
@yegor256 could be potentially useful in tests instead of `#test_log`?

Reply:
@yegor256 It was for readability and also to prevent line 45 exceed the characters limits (https://github.com/rubocop-hq/ruby-style-guide#80-character-limits). For guidelines you cannot also have multiline ternary operators ([link](https://github.com/rubocop-hq/ruby-style-guide#no-multiline-ternary)). So ok I can switch it back to the `new` methods, but then I will go for an if / else statement.

Reply:
@ledestin I think we can always expose in the future, it if we encounter a case we actually need it.